CDCgov/prime-reportstream
# Engagement/jgilmore/12693 nv receiver migration

This PR creates an HL7 receiver transform for NV and associated integration tests

Test Steps:
1. run ./prime test and ensure tests pass
2. run ./gradlew testIntegration and ensure tests pass

## Changes
- add new NV receiver HL7 transform
- Add new integration test for new transform and associated test files
- add common components to use for AOE-to-NTE mapping

### Testing
- [x ] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?

## Linked Issues
- Fixes #12771
